chore: initiate CONTRIBUTING.md (#100)
warnet-gi/WarnetBot: README.md

---

## Requirement

- python 3.10
- postgresql

## How to contribute?

> Start to contribute to our community by following this [contributing guidelines](https://github.com/Iqrar99/WarnetBot/blob/main/CONTRIBUTING.md).

Reviewer: ```suggestion
Start contributing to our community by following the [contributing guideline](https://github.com/Iqrar99/WarnetBot/blob/main/CONTRIBUTING.md).
```

Also you can change the link into a relative path so we don't have to hardcode the link anymore.
